exercise22_beyond3.py

```diff
@@ -0,0 +1,34 @@
+import csv
+import random
+
+
+def write_csv_file(filename):
+    with open(filename, 'w') as file:
+        csv_file = csv.writer(file, delimiter="\t")
+
+        for i in range(3):
+            set_of_numbers = ()
+
+            for x in range(10):
+
+                set_of_numbers = set_of_numbers + (random.randint(10, 100), )
```

Comment:
**Set** has a method **add**. So you should be able to simplify it by
set_of_numbers.add(random.randint(10, 100))

Reply:
It is possible that random number was already added to the set before, in such case you need to handle it correctly and regenerate another random number, otherwise your line gonna contain less than 10 elements.